### PR TITLE
Document how the whole integration between Gazebo, ros2_control and MoveIt2 happens

### PR DESCRIPTION
**Pull request by j-rivero**
The PR is an effort that we (Open Robotics, particularly @mabelzhang and myself) did as part of one of our projects to document the whole integration of Gazebo, ro2_control and MoveIt2.

Some details discussed during the review process are in the original PR https://github.com/j-rivero/ign_moveit2_examples/pull/1 including a discussion on using Ignition or Gazebo :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 C++ and Python examples of using MoveIt 2 for planning motions that are executed inside ~~Ignition~~ Gazebo simulation environment. These examples make use of [ros2_control](https://github.com/ros-controls/ros2_control) via [gz_ros2_control](https://github.com/ros-controls/gz_ros2_control).
 
+The interaction among different ROS 2, MoveIt 2 and Gazebo components is further documented in [docs/README.md](./docs/README.md), alongside suggestions for implementing a similar setup on your custom robot.
```
Comment (AndrejOrsula) [inline README.md:5]: I added this sentence to the main README to increase the visibility of the documentation you worked on. Please, let me know if that is okay with you.

**Review by AndrejOrsula — APPROVED** [review]
Thank you very much for your effort and for opening this PR! :smile: 

Several people have asked me about the integration of a new robot with Gazebo + MoveIt 2, so this is incredibly useful! I haven't checked, but I assume that [moveit2_tutorials](https://github.com/ros-planning/moveit2_tutorials) will eventually have a more in-depth guide and examples. But for now, this is great!


This PR looks good to me! I just added some suggestions to use permalinks for GitHub content. I also committed one addition to the main README to improve this documentation's visibility. Let me know if that's okay with you.

FYI: Before merging, I will just run it through `pre-commit` because [mdformat](https://github.com/executablebooks/mdformat) will suggest some slight modifications.

**Review by j-rivero — COMMENTED** [review]
Great! Thanks Andrej.

**Comment by AndrejOrsula** [comment]
Thank you once again for this PR!
I ran it through `pre-commit` and changed a few more relative links into permalinks.

Merging...

**Comment by j-rivero** [comment]
> I ran it through `pre-commit` and changed a few more relative links into permalinks.

I think that something went wrong and the links are now pointing to my fork :(

**Comment by AndrejOrsula** [comment]
> I think that something went wrong and the links are now pointing to my fork :(

My bad! I did not check it properly. 
Thank you for catching it! :+1: 

It should be fixed via https://github.com/AndrejOrsula/ign_moveit2_examples/commit/1c3a35859b93ee1e099530c71e4efcacc22fd18a now.

**Comment by j-rivero** [comment]
> It should be fixed via [1c3a358](https://github.com/AndrejOrsula/ign_moveit2_examples/commit/1c3a35859b93ee1e099530c71e4efcacc22fd18a) now.

great thanks! :rocket:

**Pull request merged**